emscripten.py

```diff
@@ -882,9 +882,6 @@ def create_module_wasm(sending, receiving, invoke_funcs, metadata):
   receiving += create_named_globals(metadata)
   receiving += create_fp_accessors(metadata)
   module = []
-  module.append('var asmGlobalArg = {};\n')
```

Comment:
This line looks like it makes this change no NFC.    Is `asmGlobalArg` not used at all in wasm builds?

Wouldn't we expect a code size reduction from this?

Reply:
It's never been used, correct. grep shows no remaining uses anywhere.

I think the JS optimizer just removed it, so there is no noticeable difference in optimized builds.